docs/examples/apis/axis-margin.html

```diff
@@ -0,0 +1,354 @@
+              labelBackgroundColor: 'black',
+              labelColor: 'white',
+              labelBackgroundOpacity: 0.33,
+              labelTextOpacity: 0.8,
+              colorRange: [
+                '#b2fff0',
+                '#92edd9',
+                '#71ddc3',
+                '#4ccbae',
+                '#03ba98',
+                '#006350'
+              ],
+              colorRangeGradient: true,
+            },
+            width: 72,
+          },
+          {
+            server: 'http://higlass.io/api/v1',
+            tilesetUid: 'PjIJKXGbSNCalUZO21e_HQ',
+            uid: 'bar32',
+            type: 'vertical-bar',
+            options: {
+              axisPositionHorizontal: 'right',
+              axisMargin: 10,
+              labelPosition: 'bottomLeft',
+              labelLeftMargin: 10,
+              labelBottomMargin: 10,
+              labelBackgroundColor: 'black',
+              labelColor: 'white',
+              labelBackgroundOpacity: 0.33,
+              labelTextOpacity: 0.8,
+              colorRange: [
+                '#ffdb9e',
+                '#f7c67d',
+                '#eeb25b',
+                '#e49e38',
+                '#d88a02',
+                '#704700'
+              ],
+            },
+            width: 72,
+          },
+          {
+            server: 'http://higlass.io/api/v1',
+            tilesetUid: 'PdAaSdibTLK34hCw7ubqKA',
+            uid: 'bar42',
+            type: 'vertical-bar',
+            options: {
+              align: 'top',
+              axisPositionHorizontal: 'right',
+              axisMargin: 10,
+              labelPosition: 'bottomRight',
+              labelRightMargin: 10,
+              labelBottomMargin: 10,
+              labelBackgroundColor: 'black',
+              labelColor: 'white',
+              labelBackgroundOpacity: 0.33,
+              labelTextOpacity: 0.8,
+              colorRange: [
+                '#ffafe2',
+                '#f68fcb',
+                '#eb6eb4',
+                '#e0489e',
+                '#d30089',
+                '#6d0045'
+              ],
+            },
+            width: 72,
+          },
+          {
+            chromInfoPath: '//s3.amazonaws.com/pkerp/data/hg19/chromSizes.tsv',
+            type: 'vertical-chromosome-labels',
+            position: 'top',
+            name: 'Chromosome Labels (hg19)',
+            width: 24,
+            uid: 'chroms2',
+            options: {
+              color: '#999999',
+              showMousePosition: false,
+              mousePositionColor: '#999999'
+            },
+          }
+        ],
+        center: [],
+        right: [],
+        bottom: [],
+      },
+      layout: {
+        w: 12,
+        h: 12,
+        x: 0,
+        y: 0,
+        i: 'aa',
+        moved: false,
+        static: false
+      },
+    }
+  ],
+};
+const api = hglib.viewer(
+  document.getElementById('demo'), viewConfig, { bounded: true },
+);
+</script>
+</html>
```

Comment:
Does this do anything special with the API? If not, I'd prefer to only have the json file checked in, but I could be missing something.

Reply:
I will remove this prior to merging. My only beef with the JSON file right now is that it's more work / less convenient for quick testing. Therefore, I added an intermediate html file until the PR is fully ready.